usr/src/cmd/fs.d/nfs/statd/Makefile

```diff
@@ -28,24 +28,27 @@
 # Copyright 1990-2003 Sun Microsystems, Inc.  All rights reserved.
 # Use is subject to license terms.
 #
+# Copyright (c) 2016 by Delphix. All rights reserved.
+#
 
 FSTYPE=		nfs
 TYPEPROG=	statd
 ATTMK=		$(TYPEPROG)
 
 include		../../Makefile.fstype
-CPPFLAGS +=     -D_REENTRANT -DSUN_THREADS
 
 CERRWARN +=	-_gcc=-Wno-unused-variable
+CERRWARN +=	-_gcc=-Wno-switch
```

Comment:
Any possibility of just correctly fixing these rather than gagging new warnings? What problems was this causing?